BenchmarkTests/Benchmarks/Sources/Exporter.swift

```diff
@@ -0,0 +1,26 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-Present Datadog, Inc.
+ */
+
```

Comment:
nit: a brief comment explaining the purpose of this class could be helpful in the future